docs/source/index.rst

```diff
@@ -3,6 +3,8 @@
 RADICAL-Pilot |version| Documentation
 #####################################
 
+.. py:module:: radical.pilot
+
```

Comment:
Of course, some amount of the subsequent text could be indented to become explicitly module documentation, but I don't think there is substantial utility in doing so.